content/en/docs/tasks/administer-cluster/migrating-from-dockershim/troubleshooting-cni-plugin-related-errors.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Troubleshooting CNI plugin-related errors
+content_type: task
+reviewers:
+- SergeyKanzhelev
+weight: 10
+---
+
+<!-- overview -->
+
+To avoid CNI plugin-related errors, verify that you are using or upgrading to a
+container runtime that has been tested to work correctly with your version of
+Kubernetes.
+
+For example, the following container runtimes have been tested with Kubernetes
+1.24:
+
+* containerd v1.6.4 and later, v1.5.11 and later
+* CRI-O 1.24 and later
+
+## About the "Incompatible CNI versions" and "Failed to destroy network for sandbox" errors
+
+Service issues exist for pod CNI network setup and tear down in containerd
+v1.6.0-v1.6.3 when the CNI plugins have not been upgraded and/or the CNI config
+version is not declared in the CNI config files. These issues are resolved in
+containerd v1.6.4.
+
+With containerd v1.6.0-v1.6.3, if you do not upgrade the CNI plugins and/or
+declare the CNI config version, you might encounter the following "Incompatible
+CNI versions" or "Failed to destroy network for sandbox" error conditions.
+
+### Incompatible CNI versions error
+
+If the version of your CNI plugin does not correctly match the plugin version in
+the config because the config version is later than the plugin version, the
+containerd log may/will likely show an error message on startup of a pod similar
```

Comment:
Using either of the two instead of the two since I don't think using both adds extra context to the sentence.

```suggestion
containerd log will likely show an error message on startup of a pod similar
```

Reply:
Updated. Thanks.